contracts/tasks/deploy/maci/09-vkRegistry.ts

```diff
@@ -28,7 +28,7 @@ deployment.deployTask("full:deploy-vk-registry", "Deploy Vk Registry and set key
 
     const stateTreeDepth = deployment.getDeployConfigField<number>(EContracts.VkRegistry, "stateTreeDepth");
     const intStateTreeDepth = deployment.getDeployConfigField<number>(EContracts.VkRegistry, "intStateTreeDepth");
-    const messageTreeDepth = deployment.getDeployConfigField<number>(EContracts.VkRegistry, "messageTreeDepth");
+    // const messageTreeDepth = deployment.getDeployConfigField<number>(EContracts.VkRegistry, "messageTreeDepth");
```

Comment:
let's remove commented out code

Reply:
Max messages parameter will be removed in the next milestone.